### ISSUE TITLE
Cannot find module '../index' or its corresponding type declarations.  TS2307 (Toast.d.ts); Different Source problem

### ISSUE THREAD

**Issue by dkun9039** [reactstrap/reactstrap#2009]
- components: `Toast.d.ts`
- reactstrap version `#8.7.0`

- react version `#16.3.0`

### What is happening?

I don't know why, but it is different between the source from git and the source from npm (or yarn)...
this part: 
```
import { CSSModule } from '../index';
```
but it makes error. 
`Cannot find module ~~`

### What should be happening?

atter fix like this( same as your source code from github)
```
import { CSSModule } from './index';
```
it works good.
### Steps to reproduce issue

1. 

### Error message in console

```
TypeScript error in /Users/jeongdaun/Documents/palida_partner/node_modules/reactstrap/lib/Toast.d.ts(2,27):
Cannot find module '../index' or its corresponding type declarations.  TS2307
```

### Code
```javascript
import * as React from 'react';
import { CSSModule } from '../index'; // this is the problem
import { FadeProps } from './Fade';

export interface ToastProps extends React.HTMLAttributes<HTMLElement> {
  [key: string]: any;
  tag?: React.ElementType;
  cssModule?: CSSModule;
  innerRef?: React.Ref<HTMLElement>;
  isOpen?: boolean;
  fade?: boolean;
  transition?: FadeProps;
}

declare class Toast extends React.Component<ToastProps> {}
export default Toast;

```

**Comment by kyletsang** [comment]
This was fixed after the 8.7.0 release in #2007.

@TheSharpieOne, sorry to bug you for this, but can we get a patch release to fix the TS types?

**Comment by TheSharpieOne** [comment]
@kyletsang Yeah, I'll make a new release

Also, you should be able to make releases as well. You can run the `/scripts/release` script (it assumes you have GITHUB_TOKEN set, but you can fake it `GITHUB_TOKEN=1 sh ./scripts/release`). It will determine the version bump, perform the bump, update the CHANGELOG and commit those changes to a branch with the version in it and push it to origin. Then you can create a PR form that branch to master, wait for it to pass CI and merge it.

![image](https://user-images.githubusercontent.com/664714/98037237-4492f800-1de9-11eb-8184-6e7e31ff9a40.png)

To publish it to npm, you just need to create a release (which I do via github's release UI to add the CHANGELOG part, it's really a tag, but a "release" allows for more stuff in github). Once the release/tag is made, Travis will pick that up, generate the release artifacts and publish it to npm.

**Comment by kyletsang** [comment]
Thanks for the info @TheSharpieOne - didn't realize I had those permissions.

**Comment by kyletsang** [comment]
Fixed and released in 8.7.1
